packages/bcsc-core/android/gradle.properties

```diff
@@ -1,5 +1,8 @@
-BcscCore_kotlinVersion=2.0.21
+BcscCore_kotlinVersion=1.9.20
 BcscCore_minSdkVersion=24
 BcscCore_targetSdkVersion=35
 BcscCore_compileSdkVersion=35
-BcscCore_ndkVersion=27.1.12297006
+BcscCore_ndkVersion=25.1.8937393
```

Comment:
Why the downgrade?

Reply:
To align with our app's Android setup. This prevented some weird behavior and conflicts when linting both workspaces